### PR TITLE
#675 - small WFIRST module update

### PR DESCRIPTION
**Pull request by rmandelb**
This PR is for a small WFIRST module update that I would like to merge to master before tagging v1.3.  The aberrations in an earlier optical design were too large, and incorporation of the newer design gives a PSF that behaves more nicely (e.g., it's larger in the red bands, like a diffraction-limited PSF, instead of having such large aberrations that it becomes larger in the blue bands).  I also updated the docstring to indicate the relationship between these PSFs and the files on the WFIRST project webpage.

Two other possible updates mentioned in #675 were not done, since they are a bit more work.  They will be left for a later version of GalSim.

### CODE REVIEW

File: galsim/wfirst/wfirst_psfs.py [hunk 1]
```diff
@@ -448,3 +437,37 @@ def _find_limits(bandpasses, bandpass_dict):
         if bp.blue_limit < min_wave: min_wave = bp.blue_limit
         if bp.red_limit > max_wave: max_wave = bp.red_limit
     return min_wave, max_wave
+
+def _parse_new_zernikes(infile, out_dir):
```
Comment (jmeyers314) [inline galsim/wfirst/wfirst_psfs.py]: Is there a danger that this function name will become dated?  Maybe just `_parse_zernikes()` or `_parse_zernikes_0715()` would work better?

**Comment by jmeyers314** [comment]
Looks good to me @rmandelb .  All tests pass.

File: galsim/wfirst/wfirst_psfs.py (hunk 1 above)
Comment (rmandelb) [inline galsim/wfirst/wfirst_psfs.py]: That's a good point.  The new stand-alone routine in devel/external/ has a more specific name according to your suggestion.

**Comment by rmandelb** [comment]
@rmjarvis , did you have any other comments besides that one, or are we good to merge this?

**Comment by rmjarvis** [comment]
Fine to merge as far as I'm concerned.

**Comment by rmandelb** [comment]
Merging #675.

**Pull request merged**